cmd/wallet-web/src/config/gnap-access-token.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
why config part of src

Reply:
This is as a backup if we dont find the config file in the configurable setup. 

Thats how other backup configs are in the wallet-web
<img width="759" alt="Screen Shot 2022-06-16 at 3 21 24 PM" src="https://user-images.githubusercontent.com/7862595/174148772-9f0f52a4-b93c-47ca-a7e5-1948881790e0.png">